lib/rules/at-rule-descriptor-no-unknown/__tests__/index.mjs

```diff
@@ -85,5 +97,32 @@ testRule({
 			endColumn: 47,
 			description: 'an unknown descriptor in a nested at-rule',
 		},
+		{
+			code: '@page { bar: 0; }',
+			message: messages.rejected('@page', 'bar'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 12,
+			description: 'an unknown descriptor or page context property in @page',
+		},
+		{
+			code: '@page { content: "foo"; }',
+			message: messages.rejected('@page', 'content'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 16,
+			description: 'a margin context property in @page',
+		},
+		{
+			code: '@page { @top-center { bar: 0; } }',
+			message: messages.rejected('@page', 'bar'),
```

Comment:
question

Will it warn twice?
i.e. `@top-center` also contains descriptors
e.g.
```
warnings: [
  {
    /* … */
    message: messages.rejected('@page', 'bar'),
  },
  {
    /* … */
    message: messages.rejected('@top-center', 'bar'),
  },
]
```

Reply:
It checks `@top-conter` but in the context of `@page` as CSSTree only has descriptors for `@page`: https://github.com/stylelint/stylelint/pull/8319/commits/34da558ee366c1d9d4d2dd0fc373ea65e290f555.

Reply:
https://csstree.github.io/docs/syntax/#discovery&q=ZGljdC5bdHlwZT0iQXRydWxlIl0%3D